app/controller/panel/TimeSlider.js

```diff
@@ -69,16 +70,23 @@ Ext.define('CpsiMapview.controller.panel.TimeSlider', {
     onTimeChanged: function (slider, isRange) {
         var me = this;
         var values = slider.getValues();
+        var timeIncrementUnit = slider.up('cmv_timeslider').timeIncrementUnit || 'year';
         var timeLayers = BasiGX.util.Layer.getLayersBy('isTimeDedendent', true);
-        // TODO: WFS layers must be filtered
         Ext.each(timeLayers, function (layer) {
-            if (layer && (layer.getSource() instanceof ol.source.TileWMS || layer.getSource() instanceof ol.source.ImageWMS)) {
-                var wmsLayerSource = layer.getSource();
-                if (wmsLayerSource) {
-                    wmsLayerSource.updateParams({
-                        TIME: me.formatWmsDateString(values, isRange)
+            if (layer) {
+                var layerSource = layer.getSource();
+                var dateFormat = layer.get('dateFormat') || 'C';
+                if (layerSource && (layerSource instanceof ol.source.TileWMS || layerSource instanceof ol.source.ImageWMS)) {
+                    layerSource.updateParams({
+                        TIME: me.formatWmsDateString(values, isRange, timeIncrementUnit, dateFormat)
                     });
                 }
+                if (layerSource instanceof ol.source.Vector) {
+                    var timeString = me.formatWmsDateString(values, isRange, timeIncrementUnit, dateFormat);
+                    var filterParts = BasiGX.util.WFS.getTimeFilterParts(layer, 'time', timeString);
```

Comment:
Is the second parameter of `getTimeFilterParts` always `'time'`?

Reply:
Good hint! No it isn't. In d4bdac81 I add the `timeProperty` as layer property. This will be not used for WFS layer to filter by time